QuantConnect.IBAutomater/IBAutomater.cs

```diff
@@ -899,6 +900,33 @@ private string GetIbGatewayVersionPath()
             return ibGatewayVersionPath;
         }
 
+        private string GetIbGatewayIniFile()
+        {
+            return Path.Combine(IsWindows ? GetIbGatewayVersionPath() : _ibDirectory, "jts.ini");
```

Comment:
Interesting, seems this `IsWindows ? GetIbGatewayVersionPath() : _ibDirectory` should be handled by `GetIbGatewayVersionPath` already?

Reply:
Unfortunately the ini file is written in two different locations:
- Windows: `c:\Jts\ibgateway\985\jts.ini` 
- Linux: `/QuantConnect/live/Jts/jts.ini`

GetIbGatewayVersionPath():
- Windows: `c:\Jts\ibgateway\985\`
- Linux: `/QuantConnect/live/ibgateway`